app/src/main/java/org/oppia/app/topic/review/ReviewFragment.kt

```diff
@@ -0,0 +1,24 @@
+package org.oppia.app.topic.review
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.oppia.app.fragment.InjectableFragment
+import javax.inject.Inject
+
+/** Fragment to show content for skill review. */
+class ReviewFragment: InjectableFragment(){
```

Comment:
Nit: space between ``()`` and ``{``

Reply:
Removed this file, this has been covered in `ConceptCardFragment`.